src/core-runtime.js

```diff
@@ -30,6 +30,14 @@ Globalize.locale = function( locale ) {
 	return this._locale;
 };
 
+Globalize._loadRuntimeFn = function( key ) {
+	var fn = Globalize[ key ];
+	if ( typeof fn === "function" ) {
+		fn.runtimeKey = key;
+	}
+	return fn;
+};
```

Comment:
You're right, fortunately, globalize-compiler currently has a peer dependency on globalize <1.3.0, so that'll get people to upgrade the compiler as well.